packages/twenty-front/src/modules/auth/components/Verify.tsx

```diff
@@ -28,14 +34,16 @@ export const Verify = () => {
       });
     }
 
+    if (!clientConfigLoaded) return;
```

Comment:
as discussed this may cause a flash. to be changed to the SigninupLoading componennt